actors/user/src/main/java/org/sunbird/user/util/MigrationUtils.java

```diff
@@ -44,4 +44,13 @@ public static void markUserAsRejected(ShadowUser shadowUser) {
         updateRecord(propertiesMap, shadowUser.getChannel(), shadowUser.getUserExtId());
         ProjectLogger.log("MigrationUtils:markUserAsRejected:update in cassandra  with userExtId" + shadowUser.getUserExtId(),LoggerEnum.INFO.name());
     }
+
+    public static ShadowUser getRecord(String userId,String extUserId) {
```

Comment:
is this method getting called from different places , if yes then write java doc for that , if no then make this as private and see "static" key required or not.

Reply:
done